tests/ClientTest.php

```diff
@@ -131,6 +131,8 @@ public function testBeforeNotifySkipsError()
     {
         $this->client = new Client($this->config = new Configuration('example-api-key'), null, $this->guzzle);
 
+        $this->client->setBatchSending(false);
```

Comment:
Without this, the original test would have still passed, even if we didn't skip sending!